code/modules/vending/games.dm

```diff
@@ -13,8 +13,8 @@
 		/obj/item/hourglass = 2,
 		/obj/item/instrument/piano_synth/headphones = 4,
 		/obj/item/camera = 3,
-		/obj/item/dyespray = 3)
-	contraband = list(/obj/item/dice/fudge = 9)
+		/obj/item/dyespray = 3,
+		/obj/item/dice/fudge = 9)
```

Comment:
bags of dice already have fudge dice. It would prob be fine to remove the extra one!